src/binance/bnbInfo.ts

```diff
@@ -6,11 +6,11 @@ import { BinanceSettings } from './bnbTypes'
 
 const otherSettings: BinanceSettings = {
   binanceApiServers: [
-    'https://dex.binance.org',
     'https://dex-atlantic.binance.org',
     'https://dex-asiapacific.binance.org',
     'https://dex-european.binance.org'
-  ]
+  ],
+  binanceNewApiServers: ['https://api.binance.org']
```

Comment:
These are only used to hit the tx endpoints so the name should reflect that instead of 'new'

Reply:
For now we are only using the `txs` endpoints, but the servers themselves are of a new API class. Binance hasn't given them a precise name so I can only call them "new" API servers.